content/en/docs/tasks/debug-application-cluster/resource-usage-monitoring.md

```diff
@@ -46,73 +47,61 @@ monitoring statistics by default:
 
 ### Kubelet
 
-The Kubelet acts as a bridge between the Kubernetes master and the nodes. It manages the pods and containers running on a machine. Kubelet translates each pod into its constituent containers and fetches individual container usage statistics from the container runtime, through the container runtime interface. For the legacy docker integration, it fetches this information from cAdvisor.  It then exposes the aggregated pod resource usage statistics through the kubelet resource metrics api.  This api is served at `/metrics/resource/v1alpha1` on the kubelet's authenticated and read-only ports.
+The Kubelet acts as a bridge between the Kubernetes master and the nodes. 
+It manages the pods and containers running on a machine. Kubelet translates each pod 
+into its constituent containers and fetches individual container usage statistics 
+from the container runtime, through the container runtime interface. 
+For the legacy docker integration, it fetches this information from cAdvisor.  
+It then exposes the aggregated pod resource usage statistics through the kubelet 
+resource metrics api.  This api is served at `/metrics/resource/v1alpha1` on the 
+kubelet's authenticated and read-only ports.
 
 ### cAdvisor
 
-cAdvisor is an open source container resource usage and performance analysis agent. It is purpose-built for containers and supports Docker containers natively. In Kubernetes, cAdvisor is integrated into the Kubelet binary. cAdvisor auto-discovers all containers in the machine and collects CPU, memory, filesystem, and network usage statistics. cAdvisor also provides the overall machine usage by analyzing the 'root' container on the machine.
+cAdvisor is an open source container resource usage and performance analysis 
```

Comment:
Per #15576, this section no longer meets content guidelines because it's:
* not in kubernetes or kubernetes-sigs
* not a CNCF project

So L59-69 need to be removed.

Reply:
removed